_slides/chapter1/chapter1.md

```diff
@@ -5,7 +5,7 @@
 ScPoEconometrics
 ========================================================
 author: Florian Oswald
-date: 2018-09-14
+date: 2019-07-29
```

Comment:
sorry i didnt tell you but I'm retiring this section with `_slides`. slides will be hosted in a separate repo ;-)

Reply:
Do I need to pull again from your repo to have the exact same version (except the file I'm editing) ?

Reply:
don't worry about that part - just leave as is. your version is mergeable with the master branch. i'm removing the slides on a different branch.
you can just push onto that same branch that you created this PR from.